src/apps/loan-list/utils/helpers.ts

```diff
@@ -31,10 +31,7 @@ export const removeLoansWithIds = (list: LoanType[], ids: string[]) => {
 };
 
 export const materialIsOverdue = (date: string | undefined) => {
```

Comment:
While you are at it with reducing the function you could also write:
```
export const materialIsOverdue = (date: string | undefined) =>
  dayjs().isAfter(dayjs(date), "day");

Reply:
changed